dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -7,13 +7,19 @@ LABEL maintainer="e@x80.org"
 ENV DEBIAN_FRONTEND="noninteractive"
 
 RUN apt-get update -qq && apt-get install --no-install-recommends -y -qq \
+        # Dependencies of the test-suite and of external projects
```

Comment:
This is inaccurate, for example `opam` and `gcc-multilib` are core dependencies of the image itself.

Reply:
OK I will extend the comment.